### PR TITLE
Persistent header showing Update app alert.

### LINKED ISSUE #1451: Show a persistent header when the schema_version is too new.

### PR DESCRIPTION
**Pull request by advayDev1**
Scrolling, navigation, transitions, etc works as expected.

Fixes #1451

Screen Shot 2020-09-11 at 10 52 47 AM
Screen Shot 2020-09-11 at 10 55 13 AM
Screen Shot 2020-09-11 at 10 55 06 AM
Screen Shot 2020-09-11 at 10 52 55 AM
Screen Shot 2020-09-11 at 10 52 50 AM

### CODE REVIEW

File: client/flutter/lib/components/page_scaffold/page_header.dart [hunk 1]
```diff
@@ -24,6 +25,7 @@ class PageHeader extends StatelessWidget {
     this.appBarColor,
     this.inSliver = true,
     this.trailing,
+    this.primary = true,
```
Comment (brunobowden) [inline client/flutter/lib/components/page_scaffold/page_header.dart:28]: Helpful to add comments on the purpose of `heroTag`, `inSliver`, `trailing` and `primary` as that wasn't obvious to me upon reading the code.

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
[the remark this answers is not in the document]
Comment (advayDev1) [reply .vscode/launch.json:4]: Better yet - I deleted this file :)

**Comment by advayDev1** [comment]
Please merge whenever one of you approves - Thanks.

File: client/flutter/lib/components/page_scaffold/page_scaffold.dart [hunk 3]
```diff
@@ -45,29 +47,93 @@ class PageScaffold extends StatelessWidget {
...
+  }) : super(key: key);
+
+  final double topPadding;
+  final Alert alert;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.symmetric(vertical: 16, horizontal: 24),
+      //color: Colors.red,
```
Comment (advayDev1) [inline client/flutter/lib/components/page_scaffold/page_scaffold.dart:116]: > ``
```suggestion
```

File: client/flutter/lib/main.dart [hunk 4]
```diff
@@ -128,70 +130,99 @@ class _MyAppState extends State<MyApp> with WidgetsBindingObserver {
           GlobalWidgetsLocalizations.delegate,
           S.delegate
         ],
-        routes: widget.routes,
+        routes: (widget.routes),
```
Comment (advayDev1) [inline client/flutter/lib/main.dart:133]:
```suggestion
        routes: widget.routes,
```

**Review by theswerd — APPROVED** [review]
LGTM

**Pull request merged**